core/types/receipt_test.go

```diff
@@ -518,15 +518,6 @@ func TestDeriveFieldsMPS(t *testing.T) {
 }
 
 func testReceiptFields(t *testing.T, receipt *Receipt, txs Transactions, txIndex int, receiptName string, blockHash common.Hash, blockNumber *big.Int, signer Signer) {
-	if receipt.Type != txs[txIndex].Type() {
-		t.Errorf("%s.Type = %d, want %d", receiptName, receipt.Type, txs[txIndex].Type())
-	}
-	if receipt.TxHash != txs[txIndex].Hash() {
-		t.Errorf("%s.TxHash = %s, want %s", receiptName, receipt.TxHash.String(), txs[txIndex].Hash().String())
-	}
-	if receipt.BlockHash != blockHash {
-		t.Errorf("%s.BlockHash = %s, want %s", receiptName, receipt.BlockHash.String(), blockHash.String())
-	}
```

Comment:
why are we removing this part of test code

Reply:
nice catch, I don't know, I am reverting these

Reply:
seems already reverted

Reply:
I think it is an example of conflict that could easily avoid if we organize the code better